source/domain/utils_file.py

```diff
@@ -0,0 +1,51 @@
+import networkx as nx
+
+from source.store.project_information_store import project_information_store
+
+
+def create_gml_file(graph, file_path):
+    nx.write_gml(graph, file_path)
+
+
+def import_gml_graph(file_path):
```

Comment:
If I try open a invalid .gml the software freezes. This is because this try-catch is not being fully handled. You may have to have a file-open handling in another higher-level function to be able to call the "Invalid graph" window

Reply:
Done! (It's not the best way to solve this problem, but this method will be optimized in #320).